pkg/storaged/pages.jsx

```diff
@@ -862,8 +862,17 @@ export const StorageDescription = ({ title, value, action, help, children }) =>
 export const StoragePage = ({ location, plot_state }) => {
     const page = get_page_from_location(location);
 
+    useEffect(() => {
```

Comment:
Hmm, I think I would prefer this in `storaged.jsx`, close to the `client.init()` call.

```
    componentDidMount() {
        client.addEventListener("changed", this.on_client_changed);
        cockpit.addEventListener("locationchanged", this.on_navigate);
        client.init(() => { 
            if (client.in_anaconda_mode()) {
                document.body.classList.add("anaconda");
            this.setState({ inited: true });
        });
        window.setTimeout(() => { if (!this.state.inited) this.setState({ slow_init: true }); }, 1000);
    }
```

Looking at this now, I would also prefer to move the `client.init` call to the `init` function in `storaged.jsx`, but that's something for another day.